packages/primitives/table/src/Table.tsx

```diff
@@ -24,8 +24,7 @@ export const Table = component(
       flexGrow: 1,
       flexShrink: 1,
       alignSelf: 'stretch',
-      maxWidth: '100%',
-      minWidth: 0,
+      width: '100%',
```

Comment:
why?

Reply:
Only with `table-layout: fixed` AND `width: 100%` cell stops stretching to long text and allows truncation. `maxWidth: 100%` doesn't seem to work, but I might be missing something 🤔

Reply:
should be fine then :) we don't have many use cases yet.